src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -231,7 +231,8 @@ def _submit_bulk_run(
             # upload run to cloud if the trace destination is set to cloud
             if self._config._is_cloud_trace_destination(path=run._get_flow_dir().resolve()):
                 portal_url = self._upload_run_to_cloud(run=run, config=self._config)
-                self.run_operations.update(name=run.name, portal_url=portal_url)
+                if portal_url is not None:
```

Comment:
It's not expected that there is no portal url get returned, do you have a but item where we can see more detailed logs?

Reply:
opened a bug in ADO, and will add you in related chat group for more details; i'll merge this pr first, and we can discuss more with that WI or chat group.